src/pytkdocs/parsers/docstrings/google.py

```diff
@@ -258,8 +258,8 @@ def _parse_parameters_section(self, lines: List[str], start_index: int) -> Tuple
             except (AttributeError, KeyError):
                 self.error(f"No type annotation for parameter '{name}'")
             else:
-                if signature_param.annotation is not empty:
-                    annotation = signature_param.annotation
+                # if signature_param.annotation is not empty:
+                #     annotation = signature_param.annotation
```

Comment:
Shouldn't this condition be moved in the `else` clause line 249 instead? We *do* want to use annotations if there's no type in the docstring.